omop_examples/extract_drug_treatments_from_a_molecule.ipynb

```diff
@@ -9,7 +9,9 @@
    "source": [
     "# CMS SynPuf: How to extract drug treatments containing a specific molecule\n",
     "\n",
-    "This notebook queries the [CMS SynPuf dataset](https://console.cloud.google.com/marketplace/product/hhs/synpuf?pli=1), a public synthetic patient data in OMOP. This notebook is intended to be used as an example for how to query the public OMOP dataset, and how to do basic visualizations."
+    "This notebook queries the [CMS SynPuf dataset](https://console.cloud.google.com/marketplace/product/hhs/synpuf?pli=1), a public synthetic patient data in OMOP. This notebook is intended to be used as an example for how to query the public OMOP dataset, and how to do basic visualizations.\n",
+    "\n",
+    "> If you are **previewing** this notebook from Verily Workbench, please create a cloud environment and look for this file in the `repos/terra-axon-examples/omop_examples/` directory. Instructions for creating a cloud environment are available in the workspace description."
```

Comment:
nit: I wonder if you should use `~/repos/...`
(if so, same comment on the other nbs)